### PR TITLE
Turned on fused batch norm.

### PR DESCRIPTION
**Pull request by parvizp**
Enabling fused batch norm dropped my step time from:
```INFO:tensorflow:global step 2500: loss = 4.4076 (2.17 sec/step)```
to
```INFO:tensorflow:global step 300: loss = 6.5134 (1.82 sec/step)```

### CODE REVIEW

**Comment by Zehaos** [comment]
Thanks for your contribution, I will test it in the coming days.
